snippets/modern.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Classic Holy Grail Layout
+tags: layout,beginner
+---
+
+Create a quick template  
```

Comment:
The description needs to be more descriptive (e.g. what the layout is made up of).